packages/flutter_tools/test/commands.shard/hermetic/assemble_test.dart

```diff
@@ -113,7 +113,9 @@ void main() {
     ];
     expect(
       commandRunner.run(command),
-      throwsToolExit(message: 'Error parsing assemble command: your generated configuration may be out of date')
+      throwsToolExit(
+            message:
+                'Error parsing assemble command: your --DartDefines are not base64 encoded or your generated configuration may be out of date')
```

Comment:
chore: I would add a comma at the end of the line so the formatter can automatically print the parenthesis on a new line

```suggestion
                'Error parsing assemble command: your --DartDefines are not base64 encoded or your generated configuration may be out of date',)
```